exasol/bucketfs/_error.py

```diff
@@ -6,3 +6,10 @@ class BucketFsError(Exception):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+
+class MountedBucketFsError(BucketFsError):
```

Comment:
Ideally this would be more specific than `BucketFsError` but not backend specific e.g.: `FileError`, `IoError`, `FileNotFoundError` , `IsADirectoryError`, `PermissionError` ...


Still details regarding e.g. the backend can be part of the error message.

My general take on this would be: 
ErrorType: Generic over a set of errors, ErrorMessage: As specific as possible.